connector/profiler/pom.xml

```diff
@@ -45,6 +48,7 @@
       <groupId>me.bechberger</groupId>
       <artifactId>ap-loader-all</artifactId>
       <version>3.0-8</version>
+      <scope>provided</scope>
```

Comment:
cc @parthchandra , too

Reply:
It's great to include this feature in the spark release. I feel though, that if we are making it available in the release the ap-loader dependency should be included as well. (Currently, if we build with the jvm-profiler profile, the dependency is included in the build). 
Surprisingly, the jar file is not very large (I have version 2.9.7 and it is only 830K).
Either way, including this in the release is already a big step so I'm fine with asking users to download the jar.